deployments/polygon/usdc/configuration.json

```diff
@@ -31,7 +31,7 @@
       "borrowCF": 0.825,
       "liquidateCF": 0.895,
       "liquidationFactor": 0.95,
-      "supplyCap": "0e18"
+      "supplyCap": "10_000e18"
```

Comment:
These are just placeholders until we get the recommendations from Gauntlet. These match the numbers in `verify`.